dev-docs/explanations/how_auto_attach_works.md

```diff
@@ -17,8 +17,9 @@ The `pro auto-attach` command follows a specific flow on every **Public Cloud Ub
    * **Azure**: http://169.254.169.254/metadata/attested/document?api-version=2020-09-01
    * **GCP**: http://metadata/computeMetadata/v1/instance/service-accounts/default/identity
 
-> **Note**
-> On some instances, like AWS, we can also use the IPv6 address to fetch the metadata
+```{note}
+On some instances, like AWS, we can also use the IPv6 address to fetch the metadata
```

Comment:
Is an instance a 'thing' that some other thing sits on? (in which case, this is fine and please disregard my comment). 

Or did you mean it in the sense of "in some cases"? If so, this should be "In some instances".

Reply:
`Instance` in this case refers to a running Virtual Machine - so I think it makes sense as is.
However, we could rephrase it as
```suggestion
Some instances, in AWS for example, are able to also use the IPv6 address to fetch the metadata
```
or something alike

Reply:
Just double checked and this is only a feature on AWS (for now), so I think I'll just say that outright
```suggestion
On AWS, the client will also try the IPv6 address (`[fd00:ec2::254]`) to fetch the metadata if the IPv4 address doesn't work.
```